app/controllers/trainees_controller.rb

```diff
@@ -16,6 +16,7 @@ def index
     @draft_trainees = paginated_trainees.select(&:draft?)
     @completed_trainees = paginated_trainees.reject(&:draft?)
     @training_routes = policy_scope(Trainee).group(:training_route).count.keys
+    @trainees_imported_from_apply = policy_scope(Trainee).with_apply_application
```

Comment:
Would it be better to do a count on this instead?
```suggestion
    @trainees_imported_from_apply_count = policy_scope(Trainee).with_apply_application.count
```

Reply:
done 👍